compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -717,12 +720,9 @@ public void visit(BLangRecordVariableDef varDefNode) {
         BLangRecordVariable varNode = varDefNode.var;
 
         final BLangBlockStmt blockStmt = ASTBuilderUtil.createBlockStmt(varDefNode.pos);
-
-        BType runTimeType = new BMapType(TypeTags.RECORD, symTable.anyType, null);
```

Comment:
When we have Maptype with record type tag , bir parser failed. hence replaced with symTable.mapType